package.json

```diff
@@ -1,29 +1,59 @@
 {
-  "name": "kilt-extension-api",
+  "name": "@kiltprotocol/kilt-extension-api",
   "version": "0.1.0",
   "main": "./index.js",
   "types": "./index.d.ts",
   "type": "commonjs",
   "exports": {
     ".": {
-      "types": "./index.d.ts",
+      "import": "./dist/esm/index.js",
+      "require": "./index.js",
       "default": "./index.js"
     },
     "./wellKnownDidConfiguration": {
-      "types": "./wellKnownDidConfiguration/index.d.ts",
+      "import": "./dist/esm/wellKnownDidConfiguration/index.js",
```

Comment:
Can we shorten the path to the esm build to just /esm/ ?

Reply:
updated in: [3ae03a2](https://github.com/KILTprotocol/kilt-extension-api/pull/27/commits/3ae03a28dfb736af8ff82c330a70622277a2bdfe)